src/uct/ib/ud/base/ud_ep.h

```diff
@@ -85,7 +150,10 @@ struct uct_ud_ep {
          uct_ud_psn_t           max_psn;      /* Largest PSN that can be sent - (ack_psn + window) (from incoming packet) */
          uct_ud_psn_t           acked_psn;    /* last psn that was acked by remote side */
          ucs_queue_head_t       window;       /* send window */
-         uct_ud_ep_pending_op_t pending;      /* pending control ops */
+         uct_ud_ep_pending_op_t pending;      /* pending ops */
+         ucs_time_t             send_time;    /* tx time of last packet */
+         uct_ud_psn_t           rt_psn;       /* last psn that was retransmitted */
```

Comment:
better but this psn together with other psn's to conserve memory used by field padding